documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,252 @@
+          name: my-topic
+          patternType: literal
+        operation: Read
+      - resource:
+          type: topic
+          name: my-topic
+          patternType: literal
+        operation: Describe
+      - resource:
+          type: group
+          name: my-group
+          patternType: literal
+        operation: Read
+----
+<1> The label must match the label of the Kafka cluster for the user to be created.
+<2> Authentication specified as `tls`.
+<3> Simple authorization requires an accompanying list of ACL rules to apply to the user.
+The rules define the operations allowed on Kafka resources based on the _username_ (`my-user`).
+
+. Create or modify the `KafkaUser` resource.
++
+[source,shell,subs="+quotes,attributes"]
+kubectl apply -f _USER-CONFIG-FILE_
++
+The user is created, as well as a Secret with the same name as the `KafkaUser` resource.
+The Secret contains a private and public key for TLS client authentication.
++
+For example:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-user
+  labels:
+    strimzi.io/kind: KafkaUser
+    strimzi.io/cluster: my-cluster
+type: Opaque
+data:
+  ca.crt: _PUBLIC-KEY-OF-THE-CLIENT-CA_
+  user.crt: _USER-CERTIFICATE-CONTAINING-PUBLIC-KEY-OF-USER_
+  user.key: _PRIVATE-KEY-OF-USER_
+  user.p12: _P12-ARCHIVE-FILE-STORING-CERTIFICATES-AND-KEYS_
+  user.password: _PASSWORD-PROTECTING-P12-ARCHIVE_
+----
+
+. Configure your client to connect to the Kafka cluster with the properties required to make a secure connection to the Kafka cluster.
+
+.. Add the authentication details for the public cluster certificates:
++
+[source,env,subs="+quotes,attributes"]
+----
+security.protocol: SSL <1>
+ssl.truststore.location: _PATH-TO/ssl/keys/truststore_ <2>
+ssl.truststore.password: _CLUSTER-CERT-PASSWORD_ <3>
+ssl.truststore.type=PKCS12 <4>
+----
+<1> Enables TLS encryption (with or without TLS authentication).
+<2> Specifies the truststore location where the certificates were imported.
+<3> Specifies the password for accessing the truststore. This property can be omitted if it is not needed by the truststore.
+<4> Identifies the truststore type.
++
+NOTE: Use `security.protocol: SASL_SSL` when using SCRAM-SHA authentication over TLS.
+
+.. Add the bootstrap address and port for connecting to the Kafka cluster:
++
+[source,env,subs="+quotes,attributes"]
+----
+bootstrap.servers: _BOOTSTRAP-ADDRESS:PORT_
+----
+
+.. Add the authentication details for the public user certificates:
++
+[source,env,subs="+quotes,attributes"]
+----
+ssl.keystore.location: _PATH-TO/ssl/keys/user1.keystore_ <1>
+ssl.keystore.password: _USER-CERT-PASSWORD_ <2>
+----
+<1> Specifies the keystore location where the certificates were imported.
+<2> Specifies the password for accessing the keystore. This property can be omitted if it is not needed by the keystore.
++
+The certificate is signed by the client CA when the client connects to the Kafka cluster.
```

Comment:
This sentence is misleading. The certificate is already signed by the client CA when it's created not when the client connects to the Kafka cluster.

Reply:
Changed to: _The certificate is signed by the client CA when it is created._